docs/custom.css

```diff
@@ -3,7 +3,10 @@ html {
 }
 
 body {
-    min-width: 460px;
+    /* ADDED - Add Flexbox for responsiveness */
```

Comment:
Could you please these `/* ADDED */` comments?  Git will keep track of what you added and why :wink:

Reply:
Absolutely! I actually originally left these as notes to myself and meant to remove them 😅.